src/components/Dropzone/index.vue

```diff
@@ -0,0 +1,277 @@
+<template>
+  <div
+    :id="id"
+    :ref="id"
+    :action="url"
+    class="dropzone"
+  >
+    <input
+      type="file"
+      name="file"
+    >
+  </div>
+</template>
+
+<script lang="ts">
+import Dropzone from 'dropzone'
+import 'dropzone/dist/dropzone.css'
+import { Component, Prop, Vue, Watch } from 'vue-property-decorator'
+
+export interface DropzoneFile extends File {
+  previewElement: HTMLElement
+  previewTemplate: HTMLElement
+  previewsContainer: HTMLElement
+  status: string
+  accepted: boolean
+  xhr?: XMLHttpRequest
+}
+Dropzone.autoDiscover = false
+@Component
+export default class Dropzonec extends Vue {
+  @Prop({ required: true }) private id!: string
+  @Prop({ required: true }) private url!: string
+  @Prop({ default: '' }) private clickable!: boolean | string | HTMLElement | (string | HTMLElement)[]
+  @Prop({ default: '上传图片' }) private defaultMsg!: string
+  @Prop({ default: '' }) private acceptedFiles!: string
+  @Prop({ default: 200 }) private thumbnailHeight!: number
+  @Prop({ default: 200 }) private thumbnailWidth!: number
+  @Prop({ default: true }) private showRemoveLink!: boolean
+  @Prop({ default: 2 }) private maxFilesize!: number
+  @Prop({ default: 3 }) private maxFiles!: number
+  @Prop({ default: true }) private autoProcessQueue!: boolean
+  @Prop({ default: false }) private useCustomDropzoneOptions!: boolean
+  @Prop({ default: '' }) private defaultImg!: string[]
+  @Prop({ default: false }) private couldPaste!: boolean
+  private dropzone?: Dropzone
+  private initOnce = true
+
+  mounted() {
+    const element:any = document.getElementById(this.id)
```

Comment:
Can we avoid use 'any' here? what does new Dropzone(...) need? Element? HTMLElement?

Reply:
HTMLElement | null